### ISSUE TITLE
Support rotation

### ISSUE THREAD

**Issue by domoritz** [handstuyennn/geo#9]
It would be great if this extension supported rotating latitude and longitude around the three spherical axes.

**Comment by blackrez** [comment]
Hello,

Like https://postgis.net/docs/ST_Rotate.html ?

**Comment by domoritz** [comment]
No, that just rotates the points on the surface. I want to rotate around the spherical axes (like rotating the globe so that I can center projections differently). Maybe it's what the "See Also" rotations at the bottom of your link are.
